.github/workflows/typescript_test.yml

```diff
@@ -33,6 +53,58 @@ env:
   PLAYWRIGHT_BROWSERS_PATH: "ms-playwright"
 
 jobs:
+  determine-test-suite:
+    runs-on: ubuntu-latest
+    outputs:
+      suites: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.ref || github.ref }}
+          fetch-depth: 0
+
+      - name: Paths Filter
+        id: filter
+        uses: dorny/paths-filter@v2
+        with:
+          filters: .github/filters.yml
+          base: ${{ github.event.before }}
+          head: ${{ github.sha }}
+
+      - name: Set Matrix
+        id: set-matrix
+        run: |
+          # Start with input suites if provided, otherwise empty array
+          SUITES="${{ inputs.suites || '[]' }}"
+
+          RELEASE="${{ inputs.release || 'false' }}"
+
+          if [[ "$RELEASE" == "true" ]]; then
+            SUITES='["release"]'
+          else
```

Comment:
Question: if you don't pass any tag will it run all tests? Would that work the same way as putting `release` in all tests?

Reply:
We could have done it this way, however, there are some tests that really aren't necessary and would be nice to run only in a 'release' scenario. Otherwise, we should create another category for these tests or put them together with others... So anyway, it's better to put them under the @release tag

Reply:
I see. Agree